dotnetv3/EC2/Actions/EC2Wrapper.cs

```diff
@@ -0,0 +1,707 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace EC2Actions;
+
+// snippet-start:[EC2.dotnetv3.EC2WrapperClass]
+/// <summary>
+/// Methods of this class perform Amazon EC2 actions.
+/// </summary>
+public class EC2Wrapper
+{
+    private readonly IAmazonEC2 _amazonEC2;
+
+    public EC2Wrapper(IAmazonEC2 amazonService)
+    {
+        _amazonEC2 = amazonService;
+    }
+
+    // snippet-start:[EC2.dotnetv3.AllocateAddress]
+    /// <summary>
+    /// Allocate an Elastic IP address.
+    /// </summary>
+    /// <returns>The allocation Id of the allocated address.</returns>
+    public async Task<string> AllocateAddress()
+    {
+        var request = new AllocateAddressRequest();
+
+        var response = await _amazonEC2.AllocateAddressAsync(request);
+        return response.AllocationId;
+    }
+    // snippet-end:[EC2.dotnetv3.AllocateAddress]
+
+    // snippet-start:[EC2.dotnetv3.AssociateAddress]
+    /// <summary>
+    /// Associate an Elastic IP address to an EC2 instance.
+    /// </summary>
+    /// <param name="allocationId">The allocation Id of an Elastic IP address.</param>
+    /// <parem name="instanceId">The instance ID of the EC2 instances to which the
```

Comment:
Should this be `param`?

Reply:
This text is generated by Visual Studio. These lines are documentation comments. So, yes, they should say `param`.